### PR TITLE
Backgrounds, part 2

### PR DESCRIPTION
**Pull request by istarkov**
## Description

Backgrounds part 2. ref #877

- [x] - remove repeat section under gradient part
- [x] - info Icon for gradient "code"

The floating panel, gradients support.

image

No drag, no hide, no smart insert

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

**Review by kof — APPROVED** [review]

File: packages/react-sdk/src/css/categories.ts [hunk 1]
```diff
@@ -120,8 +120,9 @@ const backgrounds = [
   "backgroundColor",
   "backgroundImage",
   "backgroundOrigin",
-  "backgroundPositionX",
-  "backgroundPositionY",
+  "backgroundPosition",
+  // "backgroundPositionX",
+  // "backgroundPositionY",
```
Comment (istarkov) [inline packages/react-sdk/src/css/categories.ts:125]: delete

File: apps/builder/app/builder/features/style-panel/sections/backgrounds/background-content.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      value: [{ type: "keyword", value: "none" }],
+    },
+  },
+});
+const deleteProperty = () => () => {
+  // do nothing
+};
+
+export const BackgroundContentStory = () => {
+  const eltRef = useRef<HTMLDivElement>(null);
```
Comment (kof) [inline apps/builder/app/builder/features/style-panel/sections/backgrounds/background-content.stories.tsx:23]: elementRef

File: packages/design-system/src/components/toggle-group.tsx [hunk 3]
```diff
@@ -21,21 +21,28 @@ export const ToggleGroupItem = styled(ToggleGroupPrimitive.Item, {
   alignItems: "center",
   justifyContent: "center",
   marginLeft: 1,
-  borderRadius: 2,
+  // borderRadius: 2,
```
Comment (kof) [inline packages/design-system/src/components/toggle-group.tsx:24]: ?

**Review by kof — APPROVED** [review]
looking good, would let @rpominov also look into stories organization and design system component changes

**Review by TrySound — APPROVED** [review]

**Pull request merged**